resize-font

```diff
@@ -35,6 +35,8 @@ my @fonts = (
   {'name' => 'boldItalicFont', 'code' => 713},
 );
 
+my @fixed = qw( 4x6 5x7 5x8 6x9 6x10 6x12 6x13 7x13 7x14 8x13 8x16 9x15 9x18 10x20 12x24 );
```

Comment:
In the [Wikipedia article](https://en.wikipedia.org/wiki/Fixed_(typeface)) there is a list that probably can be regarded as official (as much as we dare to attribute authenticity to Wikipedia). However, I took the actual list from the *xorg-fonts-misc* package -- on my Arch Linux system:

```
 $ pacman -Ql xorg-fonts-misc | grep -E '/[0-9]+x[0-9]+\.pcf'                                                                                                                                                                                                                                                                                     
xorg-fonts-misc /usr/share/fonts/misc/10x20.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/12x24.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/4x6.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/5x7.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/5x8.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/6x10.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/6x12.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/6x13.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/6x9.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/7x13.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/7x14.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/8x13.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/8x16.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/9x15.pcf.gz
xorg-fonts-misc /usr/share/fonts/misc/9x18.pcf.gz
```

or getting the same list in a distro independent manner:

```
$ fc-list | grep 'Fixed:style=(Regular|SemiCondensed)' -E | grep -v ISO                                                                                                                                                                                                                                                                            
/usr/share/fonts/misc/9x15.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/10x20.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/12x24.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/6x13.pcf.gz: Fixed:style=SemiCondensed
/usr/share/fonts/misc/5x7.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/8x16.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/7x14.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/5x8.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/8x13.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/6x10.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/9x18.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/4x6.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/6x12.pcf.gz: Fixed:style=SemiCondensed
/usr/share/fonts/misc/7x13.pcf.gz: Fixed:style=Regular
/usr/share/fonts/misc/6x9.pcf.gz: Fixed:style=Regular
```

I assumed *xorgs-fonts-misc* is a very basic X11 package and so what it delivers will be quite uniformly found across distros (or even beyond Linux -- anywhere where X11 is used).